target/linux/ath79/dts/qca9533_tplink_tl-wr802n.dtsi

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	chosen {
+		bootargs = "console=ttyS0,115200n8";
+	};
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+		label-mac-device = &eth0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_system: system {
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+		};
+	};
```

Comment:
As for keys, I'd prefer to have the full leds node in the DTSes if it's only one LED ...

Reply:
Moved everything back to .dtsi.